operations/template/alert.tf

```diff
@@ -61,21 +81,40 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "database_token_expired_a
     threshold = 1
   }
 
-  #   below tags are managed by CDC
   lifecycle {
     ignore_changes = [
-      tags["business_steward"],
-      tags["center"],
-      tags["environment"],
-      tags["escid"],
-      tags["funding_source"],
-      tags["pii_data"],
-      tags["security_compliance"],
-      tags["security_steward"],
-      tags["support_group"],
-      tags["system"],
-      tags["technical_steward"],
-      tags["zone"]
+      # Ignore changes to tags because the CDC sets these automagically
+      tags,
+    ]
+  }
+}
+
+resource "azurerm_monitor_metric_alert" "azure_4XX_alert" {
+  count               = local.non_pr_environment ? 1 : 0
+  name                = "cdcti-${var.environment}-azure-http-4XX-alert"
+  resource_group_name = data.azurerm_resource_group.group.name
+  scopes              = [data.azurerm_resource_group.group.id]
+  description         = "Action will be triggered when Http Status Code 4XX is greater than or equal to 1"
+  frequency           = "PT1M" // Checks every 1 minute
+  window_size         = "PT5M" // Every Check, looks back 5 minutes in history
+  //TBD: How frequent do we want this alert and how far do we want it to look back.
```

Comment:
The real question to me is how many 4XXs in so many minutes do we want to initiate an incident?  Like, if one of us does a test in production, and we accidentally hit the wrong URL that generates a 404, is that something worth generating an incident over?  Or what about if ReportStream calls us, they have accidentally lost their secrets for us, and so they start getting 401s?

So, to answer the question, I'm not sure how big the `window_size` should be, but it is impacted by the `threshold` below too.  I feel `threshold` should be greater than `1` at least.

Reply:
Thinking about this some more, perhaps the threshold should be 3 4XXs within an hour?  So `threshold` will be `3`, `frequency` can be `PT1M` (every minute) and `window_size` is `PT1H` (or `60M` if `1H` isn't valid?).  Thoughts?

Reply:
I'd have to check what are valid values but if I can I will set it to 1H.

Reply:
This has been updated to the recommended.